### ISSUE TITLE
[Task] Generate dummy data for new models

### ISSUE THREAD

**Issue by Pyxsys** [Pyxsys/SOEN390_19#39]
**Description**
Generate mock data for testing DB queries. The corresponding modeling tasks must be completed prior.
- [x] #34 mock bike + bike parts data
- [x] #36 mock business data
- [x] Admin login credentials

**Related Issues:** #34, #36, #44

**Comment by Pyxsys** [comment]
Need to add default/admin login credentials.

**Comment by Pyxsys** [comment]
Sales data is outside of scope for sprint. New issue will be required for next sprint.
